robots/EDUCATOR/touch.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python3
+"""
+Reverse robot if bumps into wall.
+
+This script is a simple demonstration of the touch sensor.
+"""
+
+from ev3dev.motor import MoveSteering, OUTPUT_B, OUTPUT_C
+from ev3dev.sensor.lego import TouchSensor
+
+motor_pair = MoveSteering(OUTPUT_B, OUTPUT_C)
+touch_sensor = TouchSensor()
+
+# Start robot moving forward
+motor_pair.on(steering=0, speed_pct=10)
+# Wait until robot touches wall
```

Comment:
This is very minor but adding a blank line before the start of a new comment makes things easier to read.

Reply:
Yep, I’ll fix that.